app/metrics.yaml

```diff
@@ -130,3 +130,130 @@ settings_screen:
     notification_emails:
       - android-probes@mozilla.com
     expires: "2022-08-17"
+
+tracking_protection:
+  toolbar_shield_clicked:
+    type: counter
+    description: |
+      A counter that indicates how many times a user has opened
+      the tracking protection settings panel from the toolbar.
+    bugs:
+      - https://github.com/mozilla-mobile/focus-android/issues/5057
+    data_reviews:
+      - https://github.com/mozilla-mobile/focus-android/pull/5163
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2022-08-17"
+
+  tracking_protection_changed:
+    type: event
+    description: |
+      The user has changed the setting for enhanced tracking protection.
+    bugs:
+      - https://github.com/mozilla-mobile/focus-android/issues/5057
+    data_reviews:
+      - https://github.com/mozilla-mobile/focus-android/pull/5163
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - android-probes@mozilla.com
+    expires: "2022-08-17"
```

Comment:
If you added an [event "extra"](https://mozilla.github.io/glean/book/reference/metrics/event.html#extra-metric-parameters) to this, you could track the new value that it was changed to in the "extra" info.

Reply:
I added  `isEnabled` extra